ivy/functional/backends/paddle/experimental/linear_algebra.py

```diff
@@ -116,7 +116,7 @@ def dot(
     *,
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    return paddle.dot(a, b, out=out)
+    return paddle.tensordot(a, b, axes=1)
 
 
 dot.support_native_out = True
```

Comment:
Let's remove this since `paddle.tensordot` doesn't natively support the `out` argument.
```suggestion
```